packages/central-server/src/apiV2/entityRelations/GETEntityRelations.js

```diff
@@ -6,12 +6,6 @@
 import { GETHandler } from '../GETHandler';
 
 export class GETEntityRelations extends GETHandler {
-  // TODO we should allow a table to be joined multiple times (here for child_id and parent_id)
-  customJoinConditions = {
-    entity: ['entity.id', 'entity_relation.child_id'],
-    entity_hierarchy: ['entity_hierarchy.id', 'entity_relation.entity_hierarchy_id'],
-  };
-
   // TODO add permissions - see GetDashboardRelations
```

Comment:
This TODO will be addressed in the next (and seemingly final) PR for this feature